### PR TITLE
Redis location

### PR DESCRIPTION
**Pull request by LindaKadz**
User can be saved to Redis db

### CODE REVIEW

File: lib/nailinda/user/user.ex [hunk 1]
```diff
@@ -2,10 +2,21 @@ defmodule Nailinda.User do
   @moduledoc """
    This is for all the users context
   """
+  alias Nailinda.Redis
   alias Nailinda.Repo
   alias Nailinda.User.{Doctor, Patient}
 
   def create_patient(attrs) do
+    %{"location" => location} = attrs
```
Comment (sigu) [inline lib/nailinda/user/user.ex]:
```elixir
def create_patient( %{"location" => location} = attrs ) do
.....
end
```

**Review by sigu — CHANGES_REQUESTED** [review]

File: lib/nailinda/user/user.ex (hunk 1 above)
Comment (LindaKadz) [reply lib/nailinda/user/user.ex]: Done! 

calling self() on redis makes the request time out, so I didn't work on that.

**Review by okothkongo — APPROVED** [review]

**Review by 75pollet — APPROVED** [review]

File: lib/nailinda/redis.ex [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    {:ok, %{}}
+  end
+
+  def start_redis do
+    {:ok, conn} = Redix.start_link("redis://redis:6379", port: 6379)
+    conn
+  end
+
+  def save_patient_location(long, lat, member) do
+    start_redis()
```
Comment (sigu) [inline lib/nailinda/redis.ex:22]: Starting genserver for every user created looks "disastrous" when this scales. We shold be able to start up one process for redix then use it for communication to redis.

Comment (sigu) [reply lib/nailinda/redis.ex:22]: WIll be resolved by issue #107

**Review by sigu — APPROVED** [review]

**Pull request merged**